### ISSUE TITLE
[Windows] Incredibly slow performance with process_iter and open_files

### ISSUE THREAD

**Issue by pyoor** [giampaolo/psutil#1883]
**Platform**
* Microsoft Windows 10 20H2
* psutil 5.7.3
* Python 3.8

Using psutil.process_iter on a Windows 10 virtual machine (VirtualBox) with the `open_files` attribute is incredibly slow.  Some processes take minutes to iterate over.  I have been unable to reproduce this under bare-metal but it occurs consistently for me in several VMs.  Is this an issue with psutil or something with the Windows VM?

```
for proc in process_iter(attrs=["name", "open_files"]):
  print(proc.info)
```

**Comment by giampaolo** [comment]
From https://psutil.readthedocs.io/en/latest/#psutil.Process.open_files

> Warning: on Windows this method is not reliable due to some limitations of the underlying Windows API which may hang when retrieving certain file handles. In order to work around that psutil spawns a thread to determine the file handle name and kills it if it’s not responding after 100ms. That implies that this method on Windows is not guaranteed to enumerate all regular file handles (see issue 597). Tools like ProcessHacker has the same limitation.

**Comment by pyoor** [comment]
In case someone else is facing a similar issue, it appears that the slow down was caused by using the bundled version of python included in the Windows store (Python 3.8).  Using the installed from python.org worked as expected.
